src/components/ExpenseDeletedAlert.js

```diff
@@ -0,0 +1,37 @@
+import { getUKFormattedEuros, getUKFormattedDate } from '../utils/helpers';
+
+const ExpenseDeletedAlert = ({
+  expenseDeleted,
+  setExpenseDeleted,
+  title,
+  date,
+  amount,
+}) => {
+  return (
+    <div
+      role="alert"
+      style={{ zIndex: 1 }}
```

Comment:
do we really need it? if yes, doesn't bootstrap offer any utility classes for this?

Reply:
I don't think there are any utility classes for this. In any case, it's just technical debt, so we can get rid of it.